### PR TITLE
[5.8] Added example for custom tags in the docs

### PR DESCRIPTION
**Pull request by wotta**
Because I couldn't find docs on how to edit add custom tags I reached out and got my answer from @themsaid.
I hope we can document this example for future references.

### CODE REVIEW

File: telescope.md [hunk 1]
```diff
@@ -45,7 +47,7 @@ You may use Composer to install Telescope into your Laravel project:
 After installing Telescope, publish its assets using the `telescope:install` Artisan command. After installing Telescope, you should also run the `migrate` command:
 
     php artisan telescope:install
-
+
```
Comment (themsaid) [inline telescope.md]: Please remove these spaces.

File: telescope.md [hunk 2]
```diff
@@ -63,7 +65,7 @@ If you plan to only use Telescope to assist your local development, you may inst
 After running `telescope:install`, you should remove the `TelescopeServiceProvider` service provider registration from your `app` configuration file. Instead, manually register the service provider in the `register` method of your `AppServiceProvider`:
 
     use App\Providers\TelescopeServiceProvider;
-
+
```
Comment (themsaid) [inline telescope.md]: Please remove these spaces.

File: telescope.md [hunk 3]
```diff
@@ -138,12 +140,12 @@ You may filter the data that is recorded by Telescope via the `filter` callback
     public function register()
     {
         $this->hideSensitiveRequestDetails();
-
+
```
Comment (themsaid) [inline telescope.md]: Please remove these spaces.

File: telescope.md [hunk 4]
```diff
@@ -138,12 +140,12 @@ You may filter the data that is recorded by Telescope via the `filter` callback
...
     {
         $this->hideSensitiveRequestDetails();
-
+    
         Telescope::filter(function (IncomingEntry $entry) {
             if ($this->app->isLocal()) {
                 return true;
             }
-
+
```
Comment (themsaid) [inline telescope.md]: Please remove these spaces.

File: telescope.md [hunk 5]
```diff
@@ -157,7 +159,7 @@ You may filter the data that is recorded by Telescope via the `filter` callback
 While the `filter` callback filters data for individual entries, you may use the `filterBatch` method to register a callback that filters all data for a given request or console command. If the callback returns `true`, all of the entries are recorded by Telescope:
 
     use Illuminate\Support\Collection;
-
+
```
Comment (themsaid) [inline telescope.md]: Please remove these spaces.

File: telescope.md [hunk 6]
```diff
@@ -166,12 +168,12 @@ While the `filter` callback filters data for individual entries, you may use the
     public function register()
     {
         $this->hideSensitiveRequestDetails();
-
+
```
Comment (themsaid) [inline telescope.md]: Please remove these spaces.

File: telescope.md [hunk 7]
```diff
@@ -166,12 +168,12 @@ While the `filter` callback filters data for individual entries, you may use the
...
     {
         $this->hideSensitiveRequestDetails();
-
+    
         Telescope::filterBatch(function (Collection $entries) {
             if ($this->app->isLocal()) {
                 return true;
             }
-
+
```
Comment (themsaid) [inline telescope.md]: Please remove these spaces.

**Comment by wotta** [comment]
@themsaid Sorry for the extra spaces, most likely came in the docs because I copied from Typora to Github.
All comments should be resolved now.

File: telescope.md [hunk 8]
```diff
@@ -181,7 +183,40 @@ While the `filter` callback filters data for individual entries, you may use the
         });
     }
 
+<a name="tagging"></a>
+
```
Comment (driesvints) [inline telescope.md]: Please remove this line.

```suggestion
```

File: telescope.md [hunk 9]
```diff
@@ -181,7 +183,40 @@ While the `filter` callback filters data for individual entries, you may use the
...
+            if ($entry->type === 'request') {
+                return ['status: ' . $entry->content['response_status']];
+            }
+
+            return [];
+        });
+     }
+
 <a name="available-watchers"></a>
+
```
Comment (driesvints) [inline telescope.md]: Please remove this line

```suggestion
```

File: telescope.md [hunk 10]
```diff
@@ -181,7 +183,40 @@ While the `filter` callback filters data for individual entries, you may use the
...
     }
 
+<a name="tagging"></a>
+
+## Tagging
+
+Telescope allows searching on specific tags to get a better detailed overview.
+
+<a name="tagging-adding"></a>
+
```
Comment (driesvints) [inline telescope.md]: Please remove this line.

```suggestion
```

File: telescope.md [hunk 11]
```diff
@@ -9,6 +9,8 @@
 - [Filtering](#filtering)
     - [Entries](#filtering-entries)
     - [Batches](#filtering-batches)
+- [Tagging](#tagging)
+    - [Adding custom tags](#tagging-adding)
```
Comment (driesvints) [inline telescope.md]: Please name the anchor the same as the title.

```suggestion
    - [Adding custom tags](#adding-custom-tags)
```

File: telescope.md [hunk 12]
```diff
@@ -181,7 +183,40 @@ While the `filter` callback filters data for individual entries, you may use the
         });
     }
 
+<a name="tagging"></a>
+
+## Tagging
+
+Telescope allows searching on specific tags to get a better detailed overview.
+
+<a name="tagging-adding"></a>
```
Comment (driesvints) [inline telescope.md]: Please name the anchor the same as the title.

```suggestion
<a name="adding-custom-tags"></a>
```

File: telescope.md (hunk 11 above)
Comment (wotta) [reply telescope.md]: Will keep this in mind next time!

**Comment by wotta** [comment]
@driesvints I submitted all changes that you requested. I will keep it in mind when I am submitting another PR.

**Pull request merged**